Firestore/core/src/remote/grpc_connection.h

```diff
@@ -85,6 +85,8 @@ class GrpcConnection {
   void Register(GrpcCall* call);
   void Unregister(GrpcCall* call);
 
+  static void SetClientLanguage(std::string language_header);
```

Comment:
The the variable name be `language_token`?

Reply:
Done.